### PR TITLE
Use tools/update-deps from os-autoinst via symlink

### PR DESCRIPTION
**Pull request by Martchus**

### CODE REVIEW

**Comment by perlpunk** [comment]
One problem I see is: I just added a new functionality to the script in https://github.com/os-autoinst/os-autoinst/pull/1455

When doing such changes, we need to be backwards compatible.
In this case we can just modify the dependencies.yaml in openQA before merging https://github.com/os-autoinst/os-autoinst/pull/1455, because the new entries in the YAML file are simply ignored in the old version of the script.

**Comment by perlpunk** [comment]
I think the actual symlink is missing, you only removed the script...

**Comment by Martchus** [comment]
Yes, it got lost during rebasing. Let's see whether it works now.

**Comment by perlpunk** [comment]
Btw, the os-autoinst repo also has to be cloned in the nightly dependency job...

File: .circleci/ci-packages.txt [hunk 1]
```diff
@@ -137,7 +137,7 @@ perl-Module-Implementation-0.09
 perl-Module-Pluggable-5.2
 perl-Module-Runtime-0.016
 perl-Module-Runtime-Conflicts-0.003
-perl-Mojo-IOLoop-ReadWriteProcess-0.25
+perl-Mojo-IOLoop-ReadWriteProcess
```
Comment (Martchus) [inline .circleci/ci-packages.txt:140]: This is only to temporarily fix the cache creation. The next nightly job will insert the (updated) version here.

**Comment by Martchus** [comment]
> Btw, the os-autoinst repo also has to be cloned in the nightly dependency job...

Yes, and I currently don't see how to make our current approach for having os-autoinst work for the nightly job. Maybe we should simply use Git subrepo after all.

**Comment by Martchus** [comment]
@perlpunk Like I said, using Git subrepo would likely be easier after all. So feel free to move your idea forward. I'm closing this PR for now.

**Comment by perlpunk** [comment]
I created https://github.com/perlpunk/os-autoinst-common, but this needs to be transferred to the os-autoinst org and I have no rights to do so.
Who can do it? @okurz @coolo ?

**Pull request closed without merging**